src/views/AnalysisView.vue

```diff
@@ -19,6 +19,8 @@ const props = defineProps<{
 }>();
 
 const cellFormat = ref<"best" | "cents" | "decimal">("best");
+const intervalMatrixIndexing = ref<"zero" | "one">("zero");
```

Comment:
If you want to persist the value you have to use a component prop and drill it back to the app using emits.
Now this line just creates a new local property.